### PR TITLE
fill-in-the-blanks: some cleanup and design tweaks

### PR DESCRIPTION
**Pull request by elbotho**

### CODE REVIEW

File: src/serlo-editor/editor-ui/plugin-toolbar/text-controls/hooks/use-formatting-options.tsx [hunk 1]
```diff
@@ -94,6 +94,16 @@ const registeredHotkeys = [
...
+  {
+    hotkey: 'mod+shift+`',
+    option: TextEditorFormattingOption.code,
+    handler: toggleCode,
+  },
+  {
+    hotkey: 'mod+shift+c',
+    option: TextEditorFormattingOption.code,
+    handler: toggleCode,
+  },
```
Comment (LarsTheGlidingSquirrel) [inline src/serlo-editor/editor-ui/plugin-toolbar/text-controls/hooks/use-formatting-options.tsx]: Two different hotkeys for code?

File: src/serlo-editor/editor-ui/plugin-toolbar/text-controls/utils/blank.ts [hunk 2]
```diff
@@ -26,10 +26,17 @@ export function getBlankElement(editor: SlateEditor): Blank | undefined {
...
+      match: (node) => {
+        const isHit = Element.isElement(node) && node.type === 'blank'
+        if (isHit) text = node
+        return isHit
+      },
     })
-    // TODO: Blank and also its content disappear. Instead blank content should be added back as text element (also not working in toggleMath)
+    if (text) {
+      Transforms.insertNodes(editor, { text: (text as Blank).correctAnswer })
+    }
```
Comment (LarsTheGlidingSquirrel) [inline src/serlo-editor/editor-ui/plugin-toolbar/text-controls/utils/blank.ts:39]: Nice! That solves the case when the selection is collapsed. However, does not work when you select something. But better than before, so keeping it. :)

File: src/serlo-editor/plugins/fill-in-the-blanks-exercise/blank-renderer.tsx [hunk 3]
```diff
@@ -33,7 +33,7 @@ export function BlankRenderer(props: {
       {mode === 'fill-in-the-blanks' ? (
         <input
           className={clsx(
-            'h-full resize-none rounded-full border border-brand bg-brand-50 px-2',
+            'my-0.5 h-full resize-none rounded-full border border-brand bg-brand-50 pl-2',
```
Comment (LarsTheGlidingSquirrel) [inline src/serlo-editor/plugins/fill-in-the-blanks-exercise/blank-renderer.tsx]: More space is nice but now the line spacing differs between lines with & without blanks

**Review by LarsTheGlidingSquirrel — APPROVED** [review]
We have an icon! :tada: 
Thank you! Many great suggestions. 
Feel free to address the comments I left, but I can do it as well. Just let me know. :)

File: src/serlo-editor/editor-ui/plugin-toolbar/text-controls/utils/blank.ts (hunk 2 above)
Comment (elbotho) [reply src/serlo-editor/editor-ui/plugin-toolbar/text-controls/utils/blank.ts:39]: When I select something (or everything) inside a `Blank` it works as expected. 
I assume you mean the case when the Blank and something else is selected?

Comment (LarsTheGlidingSquirrel) [reply src/serlo-editor/editor-ui/plugin-toolbar/text-controls/utils/blank.ts:39]: Ah sorry, should have explained that in more detail. I meant selecting a blank and other elements. We can find a solution in the next iteration.

**Pull request merged**